docs/_user_guide/best_practices_r.md

```diff
@@ -1,20 +1,28 @@
 
 ```{ .yaml .no-copy }
 😱 cluster_fork <- makeForkCluster(detectCores()) 😱
 ```
 
-Don't use `detectCores`.  This will ask the machine how many cores there are, and then your code will try to use them all - **don't do this on a shared environment**.  Instead, replace `detectCores()` with 4, or some other reasonably scaled amount of resources.  It's often a good idea to benchmark your code first before trying to scale up - consider [this guide](https://jstaf.github.io/hpc-r/parallel/) or [this one](https://bookdown.org/rdpeng/rprogdatascience/parallel-computation.html) to understand how your code might benefit from parallelization.
+Don't use `detectCores`. It checks the machine for the total number of available cores and attempts to use all of them. Instead, replace `detectCores()` with a fixed number, such as 4, or another reasonably scaled allocation of resources. It's also a good practice to benchmark your code first before scaling up. Consider [this guide](https://jstaf.github.io/hpc-r/parallel/){:target="_blank"} or [this one](https://bookdown.org/rdpeng/rprogdatascience/parallel-computation.html){:target="_blank"} to understand how your code might benefit from parallelization.
+
+!!! Warning
+    Using all available cores with `detectCores` can overwhelm shared systems, degrade performance for other users, and potentially lead to resource throttling or job termination. Always limit the number of cores to a reasonable amount, especially in shared or high-performance computing environments. Failing to do so may negatively impact both your work and the system's overall efficiency.
 
-You might not think something is running in parallel, but a package you're running might use parallelization "under the hood".  Check out how to [monitor usage](/_user_guide/best_practices_monitor_usage/) if you're not sure how!  If your code is running in parallel automatically, you'll need to figure out which options to pass to the functions you're calling in R to limit the number of cores.  For the [ranger](https://rdocumentation.org/packages/ranger/versions/0.15.1) package, for example, the `num.threads` option defaults to the number of cores (!!!) but you can override this manually.
+## Parallelization Under the Hood
 
-## An instructive example
+Even if you’re not explicitly running parallel code, some packages may use parallelization "under the hood." If you’re unsure, learn how to [monitor usage](/_user_guide/best_practices_monitor_usage/){:target="_blank"}. If your code runs in parallel automatically, figure out which options to pass to the functions you’re using in R to limit the number of cores. For example, in the [ranger](https://rdocumentation.org/packages/ranger/versions/0.15.1) package, the `num.threads` option defaults to the total number of cores (!!!). However, you can—and should-override this manually to prevent overuse of resources.
```

Comment:
Let's change `!!!` to `😱` to match the above?

Reply:
The two dashes with `can-and should-override` render differently, is that on purpose? If we want the big ones, let's explicitly use the "em dash" `—` to avoid this?

> you can—and should—override

Reply:
I intended to use an em dash in VS Code but didn’t realize it doesn’t render correctly. I changed it to "you can override."